test/runnable/testaa.d

```diff
@@ -1072,7 +1072,7 @@ void test10595()
         Wrap10595[int] wrap;
 
         wrap[0] = Wrap10595();
-        wrap[0] = 0;  // note: using 'alias this' to assign
+        // wrap[0] = 0;  // note: using 'alias this' to assign
```

Comment:
are you sure you should comment out this test?
if so please put a comment into the previous line explaining why it was commented out.
with reference to the PR.

Reply:
Maybe just add the deprecation message to the TEST_OUTPUT?

Reply:
I added it as a separate fail_compilation test.